conda/recipes/ptxcompiler/meta.yaml

```
@@ -3,6 +3,7 @@
 {% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
 {% set py_version=environ.get('CONDA_PY', 35) %}
```

Should we bump to Python 3.7? Though maybe this is already handled elsewhere

```suggestion
{% set py_version=environ.get('CONDA_PY', 37) %}
```

---

I think this is always set in gpuCI and we've dropped 37, so maybe going to 38 ?

---

Updated to "38"

---

Oops sorry old habits 🤦‍♂️

Thanks for the corrections. Agree with 3.8 👍